docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -137,4 +137,7 @@ docker run $dockerOpts $registry/$repository:$WMA_TAG
 docker exec -u root -it wmagent service cron start
 
 # Workaround su authentication issue (cron uses setuid via su)
-docker exec -u root -it wmagent sh -c "echo $wmaUser:$wmaUser | chpasswd"
+userStatus="$(docker exec -u root -it wmagent sh -c "passwd -S $wmaUser" | awk '{print $2}')"
+if [ "${userStatus:0:1}" == "P" ]; then
```

Comment:
@khurtado can you please add a comment above it summarizing this? For instance, what does `P` suppose to mean here? For instance, I tested it at CERN and the second column contains `L`.